docs/Install.md

```diff
@@ -12,17 +13,53 @@ Before deploying the tooling, you must have the following prepared
   * [Helm3](https://github.com/helm/helm/releases)
   * jq
 
-### Deployment Instructions
-To deploy Pelorus, run the following script from within the root repository directory
+### Deploy Long Term Storage
+
+#### Configure Storage Security
+
+To allow minio to run, add a security constraint context. Run the following command from within the root repository directory
+
+```
+oc apply -f storage/minio-scc.yaml
+```
+
+#### Deploy Object Storage for Pelorus
+
+To retain Pelorus dashboard data in the long-term, we'll deploy an instance of [minio](https://github.com/helm/charts/tree/master/stable/minio) and create a bucket called `thanos`.
+
+```
+helm install --set "buckets[0].name=thanos" \
```

Comment:
This assumes that you have a helm repo configured.  You many need to run `helm repo add stable https://kubernetes-charts.storage.googleapis.com`

Reply:
What namespace should this deploy into?  Currently, it just uses the existing namespace.